apps/web/pages/team/[slug]/book.tsx

```diff
@@ -83,6 +91,16 @@ export async function getServerSideProps(context: GetServerSidePropsContext) {
 
   const t = await getTranslation(context.locale ?? "en", "common");
 
+  // Checking if number of recurring event ocurrances is valid against event type configuration
+  const recurringEventCount =
+    (eventType.recurringEvent &&
+      eventType.recurringEvent.count &&
```

Comment:
You already set this property and check it before the double checks becomes redundant 
```suggestion
    (eventType.recurringEvent.count &&
```